discogs-labels.csv

```diff
@@ -291,3 +291,10 @@ Label ID, Label Name, Link
 303, Missile Records, https://www.discogs.com/label/303-Missile-Records
 3435, Rampant, https://www.discogs.com/label/3435-Rampant
 10112, Ozone, https://www.discogs.com/label/10112-Ozone-Recordings
+18, Soma Quality Recordings, https://www.discogs.com/label/18-Soma-Quality-Recordings
```

Comment:
This one is a duplicate, please remove the line

Reply:
done

Reply:
How can I find this playlist? I don't see it on your profile

Reply:
ha yeah there is only 50 max on the spotify profile
You can find all of them on [mirror.fm](https://mirror.fm/labels/)
https://mirror.fm/discogs/18/soma-quality-recordings/
https://open.spotify.com/playlist/6ZX0GWJpulkMuOlS0NNdnC?si=6e9dcc7f622a4843

Reply:
Big thanks!